app/views/CreateChannelView/RoomSettings/SwitchItemReadOnly.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { memo } from 'react';
+import { shallowEqual } from 'react-redux';
+
+import { SwitchItem } from './SwitchItem';
+
+export const SwitchItemReadOnly = memo(
```

Comment:
I wonder the real efficiency of it, first because it has a function as a prop and another because this component needs to be re-rendered and doesn't have much logic to be memorized.